django/contrib/auth/validators.py

```diff
@@ -23,3 +30,38 @@ class UnicodeUsernameValidator(validators.RegexValidator):
         'numbers, and @/./+/-/_ characters.'
     )
     flags = 0
+
+    def help_text(self):
+        return _(
+            'Enter a valid username. This value may contain only letters, '
+            'numbers, and @/./+/-/_ characters.'
+        )
+
+
+@deconstructible
+class UsernameMinimumLengthValidator:
```

Comment:
I'm unsure whether there's a good reason for this validator, since `django.core.validators.MinLengthValidator` (https://docs.djangoproject.com/en/3.1/ref/validators/#minlengthvalidator) exists already. 

I know there's a MinimumLengthValidator in the `password_validation.py` module but maybe that one is a bit redundant too.

Reply:
Thank you for the feedback. Apologies for extreme delay in response.

I have made suggested changes other than moving validators to username_validation.